src/rules/no-commented-tests.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { getDocsUrl } = require('./util');
+
+const message = 'Some tests seem to be commented';
+
+function hasAssertions(node) {
+  return /x?(test|it|describe)((\.skip|\[['"]skip['"]\]))?\(/.test(node.value);
```

Comment:
I didn't bother with matching quotes here. I think this will be rare enough and the rule is fuzzy as it is.